alacritty_terminal/src/tty/unix.rs

```diff
@@ -5,23 +5,30 @@ use std::fs::File;
 use std::io::{Error, ErrorKind, Result};
 use std::mem::MaybeUninit;
 use std::os::unix::io::{AsRawFd, FromRawFd, RawFd};
+use std::os::unix::net::UnixStream;
 use std::os::unix::process::CommandExt;
 use std::process::{Child, Command, Stdio};
+use std::sync::Arc;
 use std::{env, ptr};
 
 use libc::{self, c_int, winsize, TIOCSCTTY};
 use log::error;
-use mio::unix::EventedFd;
 use nix::pty::openpty;
 #[cfg(any(target_os = "linux", target_os = "macos"))]
 use nix::sys::termios::{self, InputFlags, SetArg};
 use signal_hook::consts as sigconsts;
-use signal_hook_mio::v0_6::Signals;
 
 use crate::config::PtyConfig;
 use crate::event::{OnResize, WindowSize};
 use crate::tty::{ChildEvent, EventedPty, EventedReadWrite};
 
+// We use the same interest for reading and writing.
+pub(crate) const PTY_READ_TOKEN: usize = 0;
+pub(crate) const PTY_WRITE_TOKEN: usize = 0;
```

Comment:
Why are they separate constants then? Why not make it a `PTY_READ_WRITE_TOKEN`?

Also couldn't this just be an enum to avoid accidental conflicts?

Reply:
They have to be separate in Windows but not Unix. They need to be `usize`, since that's the token type `polling` takes.

Reply:
+1